lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -329,7 +330,16 @@ def options_replacement_map
 
     sig { returns(T::Array[String]) }
     def device_config_commands
-      @device_infos.map(&:config_commands).flatten.compact
+      filter_commands(@device_infos.map(&:config_commands).flatten.compact)
+    end
+
+    def filter_commands(commands)
+      commands.reject do |cmd|
+        if cmd.include?('@swtpm_setup_bin@') && @run_opts[:reuse_tpm]
+          @logger.warn('Skipping SWTPM installation')
+          true
+        end
+      end
```

Comment:
I don't agree with filters in general. When we bring up the previous session all config commands were executed, so we should skip all of them.